doc/source/how-to/manage-protobuf-definitions.rst

```diff
@@ -0,0 +1,163 @@
+Manage Protobuf definitions
+===========================
+
+Protobuf service definitions provide the API specification for underlying
+server implementations, so that each consuming client library has a clear
+contract for gRPC data messages. Ideally, the .proto files have a single
+repository established as the source of truth for the .proto files,
+organized by API version increment as the API definition expands and changes.
+Since most client libraries are custom implementations enhancing the developer
+experience when consuming the service, releasing the Protobuf definitions
+publicly gives full flexibility to the developer to operate at the abstraction
+layer they choose to do so.
+
+Maintain API definition repository
+----------------------------------
+
+The Protobuf definition of the service is language agnostic, so the repository
+containing the Protobuf files can be created within the top-level Ansys
+GitHub organization. Every update of the Protobuf files follows a standard
+pull request process as a sanity-check for API definition accuracy. Language
+specific packages can be generated for each merge or on a set cadence.
+
+Managing Protobuf definitions for Python clients
+------------------------------------------------
+
+Building Python stub classes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add a python project definition to the API repository that can be utilized
+within the CI/CD build pipeline.
+
+A common tool to help build python stub classes is available and maintained
+within Ansys:
+
+`ansys-tools-protoc-helper <https://github.com/ansys/ansys-tools-protoc-helper/>`_
+
+Example inclusion within pyproject.toml build dependency configuration:
+
+.. code-block:: toml
+
+    [build-system]
+    requires = ["setuptools >= 42.0.0", "wheel", "ansys_tools_protoc_helper"]
+
+Basic python project build step that can be executed locally or within a
+build pipeline:
+
+.. code-block:: python
+
+    pip install -U pip
+    pip install build
+    python -m build
+
+Publishing Python API package
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PyPI is the common package manager where API packages are released.
+
+Example nightly build pipeline publishing the python stub package:
```

Comment:
I'd include the contents of `.github/workflows/ci.yml` rather than the nightly build action. All projects will need something like ci.yml but not necessarily a nightly build.